addons/escoria-core/game/scenes/sound/esc_music_player.gd

```diff
@@ -44,10 +44,7 @@ func set_state(p_state: String, p_force: bool = false) -> void:
 			resource.loop_end = resource.mix_rate * resource.get_length()
 		elif "loop" in resource:
 			resource.loop = true
-		if escoria.project_settings_manager.has_setting(escoria.project_settings_manager.MUSIC_VOLUME):
-			stream.volume_db = escoria.project_settings_manager.get_setting(
-				escoria.project_settings_manager.MUSIC_VOLUME
-			)
+		stream.volume_db = escoria.settings.music_volume
```

Comment:
I'm confused about this. Isn't using `escoria.project_settings_manager` the correct way or did I miss something? (probably?)

Reply:
I was actually going to ask the same thing, so maybe we both missed something.

Reply:
Fixed